client/src/components/Query/style.scss

```diff
@@ -143,10 +143,10 @@ div.query-container {
     cursor: pointer;
     padding-left: 4px;
   }
-}
 
-.ant-tag-has-color.subquery span {
-   color: #000000;
+  .anticon-close{
+    color:black;
```

Comment:
@claudia1296 Works! Did you know that in much older browsers using a name instead of the hex code did not work?